src/components/CaseStudies/AuthorBio.tsx

```diff
@@ -1,23 +1,62 @@
-export const AuthorBio: React.FunctionComponent<{
-    customer: string
+interface Author {
     image?: string
-    author: string
+    name: string
     title: string
+}
+
+export const AuthorBio: React.FunctionComponent<{
+    customer: string
     about: string
-}> = ({ customer, image, author, title, about }) => (
-    <div className="d-flex flex-column flex-md-row align-items-start">
-        {image && (
-            <div className="col-md-3">
-                <img className="rounded-circle max-w-150" src={image} alt={author} />
+    authors: Author[]
+}> = ({ customer, about, authors }) => (
+    <section>
+        {authors.length > 1 ? (
+            // Multi author design
+            <div>
+                <section className="col-lg-6 mx-auto p-0 max-w-650">
+                    <h5 className="mb-4 text-xl font-weight-bold">About {customer}</h5>
+                    <p>{about}</p>
+                </section>
+
+                <section className="d-md-flex d-block justify-content-between col-lg-9 mx-auto mt-6 p-0">
+                    {authors.map(a => (
+                        <div key={a.name}>
+                            <img
+                                className="rounded-circle max-w-150"
+                                width={150}
+                                height={150}
+                                src={a.image}
+                                alt={a.name}
+                            />
+                            <h3 className="font-weight-normal">{a.name}</h3>
```

Comment:
Looks like the Figma has some more spacing between the thumbnail and the header so we'll probably need to add a top margin here

Reply:
Good eye! Both margins are addressed in [a8dd2ef](https://github.com/sourcegraph/about/pull/5655/commits/a8dd2ef59eadfa66fd31e87fae5b5e3ab95019db) 😄